apps/email/js/config.js

```diff
@@ -0,0 +1,72 @@
+/*global requirejs, startupOnModelLoaded, TestUrlResolver */
+'use strict';
+
+// Set up loading of scripts, but only if not in tests, which set up
+// their own config.
+if (typeof TestUrlResolver === 'undefined') {
+  requirejs.config({
+    // waitSeconds is set to the default here; the build step rewrites
+    // it to 0 in build/email.build.js so that we never timeout waiting
+    // for modules in production. This is important when the device is
+    // under super-low-memory stress, as it may take a while for the
+    // device to get around to loading things email for background tasks
```

Comment:
Fixed in comment commit: 7d8f41f19303f833552bc490fe6bf87e1349526a. I also spread the comments out to reach the full 80 character width.